Gruntfile.js

```diff
@@ -237,14 +237,14 @@ module.exports = function(grunt) {
         configCallback(karmaConfig);
         karmaConfig.client.testharnessTests = require('./test/web-platform-tests-expectations.js');
         karmaConfig.client.testharnessTests.testURLList = testFiles;
-        karmaConfig.proxies['/polyfill.js'] = '/base/' + task.target + '.min.js';
+        karmaConfig.proxies['/base/polyfill.js'] = '/base/' + task.target + '.dev.js';
```

Comment:
Don't we want to be testing the minified version of the polyfill, instead of the dev version?